src/ray/gcs/format/gcs.fbs

```diff
@@ -49,6 +52,11 @@ table ObjectTableData {
   is_eviction: bool;
   // The number of times this object has been evicted from this node so far.
   num_evictions: int;
+  // Whether this object was created by ray.put.
+  is_put: bool;
+  // The task that created this object.
+  task_id: string;
```

Comment:
This shouldn't be necessary because the task ID is encoded in the object ID.

Reply:
Indeed, the task id is computed from the object id in `ObjectDirectory`. How do you propose we compute the task id in `monitor.py`?

Reply:
I think we want to expose this function to Python https://github.com/ray-project/ray/blob/a8517cc82a35e72f085884f8c9e3f9c7fb63f178/src/ray/id.h#L81

Reply:
Should this be exposed via `local_scheduler_extension.cc`?